lua/luasnip/loaders/from_snipmate.lua

```diff
@@ -208,6 +208,10 @@ function M.lazy_load(opts)
 	end
 end
 
+function M.edit_snippet_files()
+	loader_util.edit_snippet_files(cache.ft_paths)
+end
```

Comment:
I'd like to provide a better default here.
It'd be great to substitute the paths to packer with something shorter.
Is there a env-var that is set to the path packer uses?
I use a simple 
```lua
function(path, _)
	local clean_path, _ = path:gsub("/home/simon/.local/share/nvim/site/pack/packer/start", "$PACKER"):gsub("/home/simon/.config/nvim", "$NVIM")
	return clean_path
end
```
which would be a nicer default.

Reply:
Added a better default in 680e559